drivers/lsm6dsl/include/lsm6dsl-internal.h

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Done, but a lot of other drivers have a dash in the `xxx-internals.h` file name.